package.json

```diff
@@ -24,7 +24,8 @@
     "build-ts": "tsc --build",
     "build-clean": "tsc --build --clean && rm -rf 'packages/**/src/{dist,esm,bundle}' && lerna run build-clean --parallel",
     "build-demo": "lerna run build-demo",
-    "test": "yarn run lint && yarn run check && yarn run build && yarn run testonly && lerna run build-bundles && yarn run e2e",
+    "build-bundles": "lerna run build-bundles",
+    "test": "yarn run lint && yarn run check && yarn run build && yarn run testonly && yarn build-bundles && yarn run e2e && yarn build-demo",
```

Comment:
add `yarn build-demo` to our CI script.

this will ensure that packages consuming graphiql as a webpack module will not run into build errors at least.